x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_disposed_creds_details.tsx

```diff
@@ -55,51 +55,63 @@ export const AuditdDisposedLine = pure<Props>(
     session,
   }) => (
     <EuiFlexGroup justifyContent="center" gutterSize="none" wrap={true}>
-      <TokensFlexItem grow={false}>{i18n.SESSION}</TokensFlexItem>
-      <TokensFlexItem grow={false}>
+      <TokensFlexItem grow={false} component="span">
+        {i18n.SESSION}
+      </TokensFlexItem>
+      <TokensFlexItem grow={false} component="span">
         <DraggableBadge
-          id={`auditd-disposed-creds-${id}`}
+          eventId={`auditd-disposed-creds-${id}`}
```

Comment:
should this be the actual event `_id`, passed down from props?

Reply:
It is if you look below you should see the id like so:

```ts
    const id = data._id;
```

Reply:
I was under the impression that `DraggableBadge` will create a unique draggable ID by combining `eventId` `field` and `value`. (As an aside, perhaps a `contextId: string` might also make sense as an additional prop to `DraggableBadge`.)

If that's correct, should `eventId` be given the raw value of `id`, like this?:
```
eventId={id}
```

Reply:
I'm compensating for a contextId by the concatenation of the string with `auditd-disposed-creds` at the moment.

Reply:
Per conversations, I will change this to use a contextId for the contract to keep usage clear.

Reply:
Per our out of band discussion, the intent of the prefix is effectively create a `contextId: string`, which we agreed makes sense as its own prop.